tests/integration/pilot/common/routing.go

```diff
@@ -79,52 +79,48 @@ spec:
 
 func virtualServiceCases(apps *EchoDeployments) []TrafficTestCase {
 	var cases []TrafficTestCase
-	callCount := callsPerCluster * len(apps.PodB)
 	// Send the same call from all different clusters
-	for _, podA := range apps.PodA {
-		podA := podA
-		cases = append(cases,
-			TrafficTestCase{
-				name: "added header",
-				config: `
+
+	cases = append(cases,
+		TrafficTestCase{
+			name: "added header",
+			config: `
 apiVersion: networking.istio.io/v1alpha3
 kind: VirtualService
 metadata:
   name: default
 spec:
   hosts:
-  - b
+  - {{ (index .dst 0).Config.Service }}
```

Comment:
nit: is it possible to add a function on `echo.Instances` for .Service()? Then we can access it by `dst.Service`

Reply:
since this is super common